### PR TITLE
Various little fixes seen as a result of django-composer

### PR DESCRIPTION
**Pull request by hedleyroos**

### CODE REVIEW

File: formfactory/models.py [hunk 1]
```diff
@@ -133,9 +133,6 @@ class Meta(object):
     def __unicode__(self):
         return self.title
 
-    def absolute_url(self):
```
Comment (qoda) [inline formfactory/models.py]: Used by restframework_extras, should be used as action for API consumer.

Comment (hedleyroos) [inline formfactory/models.py]: Where is DRFE using that? I wrote DRFE and it doesn't.

Comment (qoda) [inline formfactory/models.py]: https://github.com/praekelt/django-formfactory/blob/develop/formfactory/api/__init__.py#L28

Comment (hedleyroos) [inline formfactory/models.py]: That's your implementation of an API. Use get_absolute_url like all our other apps.

Comment (hedleyroos) [inline formfactory/models.py]: I do understand that absolute_url makes for a nice property but the issue is the FED guys already use get_absolute_url in the other apps. This introduces ambiguity.

**Pull request merged**